nix/master.nix

```diff
@@ -266,6 +266,17 @@ in
         default = null;
         example = "/var/www/buildbot/nix-outputs";
       };
+
+      jobReportLimit = lib.mkOption {
+        type = lib.types.nullOr lib.types.ints.positive;
```

Comment:
> report builds individually as long as the number of builds is less than or equal to `n`

Is zero an option here to only use `nix-build-combined` and never report an individual build?

Reply:
Setting it to `0`  works, I'll push it after finishing the rest of the tests

Reply:
Thanks.